cdap-hbase-compat-base/src/main/java/co/cask/cdap/replication/TableUpdater.java

```diff
@@ -44,11 +44,23 @@
   public final Configuration conf;
   public boolean tableExists = false;
 
+  private long delay = 0;
```

Comment:
make these local variables (pass them into setupTime()), since they are used once and never change.